homeassistant/components/bang_olufsen/media_player.py

```diff
@@ -94,11 +94,12 @@ async def async_setup_entry(
     async_add_entities(new_entities=[BangOlufsenMediaPlayer(config_entry, data.client)])
 
 
-class BangOlufsenMediaPlayer(MediaPlayerEntity, BangOlufsenEntity):
+class BangOlufsenMediaPlayer(BangOlufsenEntity, MediaPlayerEntity):
     """Representation of a media player."""
 
-    _attr_has_entity_name = False
     _attr_icon = "mdi:speaker-wireless"
+    _attr_name = None
```

Comment:
Wouldn't this be better to have in the BangOlufsenEntity class?

Reply:
I usually only put this in the platform it applies to, so whenever someone is going to add sensors or something like that, they don't have to refactor that